### PR TITLE
evansmurithi.cloak version 0.2.0

### PR DESCRIPTION
**Pull request by superusercode**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109429&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]
Hitting VC redist error (Hyper-V VM):
image

cc @stephengillie

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

waiting until dependency support is out of experimental features and is enabled in pipelines

File: manifests/e/evansmurithi/cloak/0.2.0/evansmurithi.cloak.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cloak.exe
+    PortableCommandAlias: cloak
+  InstallerSha256: B4AE3DEA20EA9600B6D3F608E9BA1746E6D1A6AD2B66ECE2918695C64A3468D1
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```
Comment (stephengillie) [inline manifests/e/evansmurithi/cloak/0.2.0/evansmurithi.cloak.installer.yaml:26]:
```suggestion
```

File: manifests/e/evansmurithi/cloak/0.2.0/evansmurithi.cloak.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    PortableCommandAlias: cloak
+  InstallerSha256: A0354613F17BB1DACD2D41A29650296BB96EE81CD5038388EC6F2C7389D71933
+- InstallerUrl: https://github.com/evansmurithi/cloak/releases/download/v0.2.0/cloak-v0.2.0-i686-pc-windows-msvc.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cloak.exe
+    PortableCommandAlias: cloak
+  InstallerSha256: B4AE3DEA20EA9600B6D3F608E9BA1746E6D1A6AD2B66ECE2918695C64A3468D1
```
Comment (stephengillie) [inline manifests/e/evansmurithi/cloak/0.2.0/evansmurithi.cloak.installer.yaml:22]:
```suggestion
  InstallerSha256: B4AE3DEA20EA9600B6D3F608E9BA1746E6D1A6AD2B66ECE2918695C64A3468D1
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  - PackageIdentifier: Microsoft.VCRedist.2015+.x86

```

File: manifests/e/evansmurithi/cloak/0.2.0/evansmurithi.cloak.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```
[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/e/evansmurithi/cloak/0.2.0/evansmurithi.cloak.installer.yaml:29]: Apologies for missing this earlier.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by stephengillie — APPROVED** [review]

**Pull request merged**